Lib9c/Model/State/StakeState.cs

```diff
@@ -54,6 +54,7 @@ public void Achieve(int level, int step)
         public const long RewardInterval = 50400;
         public const long LockupInterval = 50400 * 4;
         public const long StakeRewardSheetV2Index = 6_700_000L;
+        public const long CurrencyAsRewardStartIndex = 6_910_000L;
```

Comment:
I cannot find the source of this index. If this is new, it should be bigger than current mainnet index (6974002).

Reply:
My understanding is that the context in which this value is used has not yet been executed.🤔